docs/dbaas/DBaaS_tepmplate.md

```diff
@@ -1,3 +1,139 @@
+# DatabaseCluster Templates
+
+DatabaseCluster Template allows operators to customize Database Clusters based on their requirements, environments, or infrastructure.
+
+Examples
+
+- **Data clusters**: Different load patterns, such as simple reads, heavy writes, 50%/50% read/write, and the number of connections.
+
+- **Infrastructure** - Different parameters and tuning for the resulting cluster: such as network configuration (load balancing, exposure), storage classes/types, etc.
+
+- **Environment**: Combination of database cluster and infrastructure that will affect the database cluster configuration.
+
+
+# Customize PXC DB configuration
```

Comment:
We should state that this is only an example

Reply:
Done